alfalfa_worker/lib/run_manager.py

```diff
@@ -150,74 +148,6 @@ def checkout_run(self, run_id: str) -> Run:
         run.dir = run_path
         return run
 
-    def add_site_to_mongo(self, haystack_json: dict, run: Run):
```

Comment:
Is there any data left being pushed into a Mongo document? I don't see it. Can we remove Mongo entirely?